week-1_additional_resources/terminal-commands.md

```diff
@@ -0,0 +1,12 @@
+# Command Line Cheat Sheet
+
+| Command | Description |
+| ----------- | ----------- |
+| mkdir *foldername* | Create a new folder with the given name |
+| cd *foldername* | Change into directory with the given name/path |
+| cd .. | Move back one directory |
+| ls | Look at the items in the current directory |
+| touch *filename* | Make a new file in current directory |
+| rm *filename* | Delete folder with given name/path |
+| rm -rf *foldername* | Delete folder with the given name/path |
```

Comment:
This looks great Chris! just wanted to request one thing.  Can you have the word folder name wrapped in angel brackets <>.  For example: mkdir <*foldername*>